mkdocs.yml

```diff
@@ -41,6 +40,13 @@ nav:
     - using_eessi/setting_up_environment.md
     - using_eessi/basic_commands.md
     - using_eessi/eessi_demos.md
+  - Contributing software to EESSI:
```

Comment:
"Contributing" is maybe not the best wording? How about "Adding ..."? Then the first item should be renamed.

It could be something like
- Adding software to EESSI
  - Overview
  - Contribution policy (requires #108)
  - Opening a pull request (anyone)
  - Building software (maintainers)
  - Debugging build issues (anyone)
  - Deploying built packages (maintainers)

Reply:
One thing I'm wondering: from 'Adding software to EESSI', is it immediately clear that this is about _anyone_ being able to do that? From that header, it could also be a page that just describes how _we_ add software to EESSI.

The nice thing about the word `contributing` is that it is immediately clear that any contributor can do that.

I do like your other headers, and the suggestion to include the contribution policy under this new header.

Reply:
Never mind, I'm bikeshedding. Adding is fine, I'll do that :D

I've created this structure, with 'coming soon' pages for the things we don't have content for yet. At least the full structure is in place like this.